Menu.js

```diff
@@ -57,16 +59,16 @@ const buildMenu = (mainWindow, loadInit) => {
             {
                 label: 'Split Open...',
                 click: (item, focusedWindow) => {
```

Comment:
Maybe to simplify, we could just handle the `Tab` case with  this PR? That case seems more compelling to me, as opening a bunch of files in a split would be somewhat unwieldy. And it looks like the `Tab Open` case works well - I think this parallels pretty well with functionality in something like VSCode.